### DISCUSSION TITLE
low delay

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [savonet/liquidsoap#2637] [General]
Hello, is there something in Liquidsoap that I can configure that allows me to decrease the live connection delay?
I realize that in centova cast has this option, in Liquidsoap I didn't find anything like it.

![](https://cdn.discordapp.com/attachments/647149735991443467/1021965779735937104/Screenshot_1293.png)

**Comment by toots** [comment]
I believe that this settings is more for the encoder's internal buffer than for the connection delay itself.

**Comment by Marcioghiraldelli** [comment]
I didn't quite understand your answer.
In this case, this configuration would be done in a player that runs the radio or internal in liquidsoap or even in an icecast server?

**Reply by toots** [reply]
The low delay settings that you pointed out is for the encoder. The encoder also has internal delays: when receiving data, it needs to buffer and make some computation before it can output the first encoded data. This settings asks the encoder to reduce this delay as much as possible

**Comment by gunsar** [comment]
> Hello, is there something in Liquidsoap that I can configure that allows me to decrease the live connection delay? I realize that in centova cast has this option, in Liquidsoap I didn't find anything like it.
> 
> 

@Marcioghiraldelli 

Can you help me?
my centova can't show the encoder options menu
even though installing liquidsoap was successful
Please help

![212504716-f8c9e8d3-a031-4218-9ca6-c134d55a6b11](https://user-images.githubusercontent.com/56213562/214976658-c922dca2-18af-4734-8ab2-83cd24d5094b.jpg)
